packages/@ama-sdk/core/src/fwk/api.interface.ts

```diff
@@ -16,3 +16,5 @@ export interface Api extends ApiName {
 export interface ApiConstructor<T extends Api = Api> extends ApiName {
   new (client: ApiClient): T;
 }
+/** Type that is the same as the `Partial<T>` type, but with a certain property excluded. */
```

Comment:
comment is unclear, the way I understand it is that the property K would not be part of the type while you have a mandatory K.